### DISCUSSION TITLE
libgcrypt-1.9.0 contains a security flaw and shouldn't be used

### DISCUSSION THREAD

**Discussion by chr4** [Homebrew/discussions#642] [Everyday usage]
Updating today, I noticed that homebrew provides (and automatically updates) to `libgcrypt-1.9.0`, a dependency used by e.g. `gnupg`.

The mentioned version [shouldn't be used according to its maintainer](https://lists.gnupg.org/pipermail/gnupg-announce/2021q1/000455.html), as it contains serious security flaws and shouldn't be used. There's a patched version `1.9.1` [available](https://lists.gnupg.org/pipermail/gnupg-announce/2021q1/000456.html).

I think version `1.9.0` should be ommited by Homebrew. Wasn't sure where to put this, as I felt it didn't fit into the "bugs" or simple "upgrade" categories.

**Comment by SMillerDev** [comment]
We already shipped the patched version about an hour ago.

**Reply by chr4** [reply]
Nice, that was quick. I just noticed, too: https://formulae.brew.sh/formula/libgcrypt

Thanks!
